classes/DailyTask.php

```diff
@@ -243,7 +264,8 @@ public static function generateTask(int $user_rank_num, string $task_activity, s
             $rep_reward_mod += UserReputation::DAILY_TASK_PVP_WIN_MOD;
         }
 
-        $difficulty_multiplier = 1;
+        $base_multiplier = mt_rand($user_rank_num, $user_rank_num + 2);
```

Comment:
We could move these base multipliers for each type of activity to Currency class as well so it's easy to see/change them all in one place and compare "specials gives 10x, arena gives 2x, special takes 50x as long as arena, maybe we give it more" etc

Reply:
I was thinking similar